content/_en/partners/get-started.md

```diff
@@ -0,0 +1,64 @@
+---
+layout: partners/get-started
+permalink: /partners/get-started/
+title: >- 
+    # Get started
+body: >-
+    Below are the steps to becoming a partner with Login.gov and an overview of the Interagency Agreement (IAA) process. During the process, we provide a free sandbox environment to create test integrations between Login.gov and your applications. Anybody with an email address ending in .gov or .mil can create an account in the sandbox environment. If you are a government contractor, ask your agency partner to help you gain access. 
+ssa_quote: >-
+    “The SSA UXG team appreciates the willingness of the Login.gov team to listen to our goals, concerns, and questions, then work with us to identify a path forward that will benefit the customers (end users) of SSA and other federal agencies. The team’s flexibility and willingness to hear our suggestions is very much appreciated. We enjoy brainstorming together and sharing research findings as we try to help each other move forward.”
+
+
+    Elizabeth Hanst
+
+    Technology Lead User Experience Specialist
+
+    Social Security Administration
+iaa_tips: >-
+    Tips for completing the IAA process:
+
+    * Secure a federal Authority To Operate (ATO)
+
+    * Have approved funding access based on the cost estimator
+
+    * Ensure your technical team has capacity for integration or plan to hire a team
+
+
+    The above suggestions may help keep your agency on track for creating the IAA and receiving proper approvals within the anticipated six to eight week timeline, and launching your integration with Login.gov. 
+partnership_steps_title: >-
+    ## Steps to partnership
+partnership_steps_header1: >-
+    Introductions and determine compatibility
+partnership_steps_body1: >-
+    <p style="display:inline">During the introduction call, an account manager will walk through Login.gov services and answer any questions that you have. This step will determine if Login.gov is a good fit for your agency.</p>
+partnership_steps_header2: >-
+    Estimate usage and test integrations
+partnership_steps_body2: >-
+    During onboarding, an account manager will work with your agency to estimate usage and provide a cost estimator. Once you’ve created your application and implemented an identity protocol, you can register it in the test environment dashboard and start testing. We advise at least two to three weeks for your team to test and integrate with Login.gov. <a target="_blank" href="https://developers.login.gov/testing/#how-to-get-started" class="external-link">Learn more about the sandbox environment</a>
+partnership_steps_header3: >-
```

Comment:
We actually don't need `>-` unless we are adding markup to the text.

Reply:
Ah, I thought it was for anything with multiple lines.